.github/workflows/vercel-pr-preview.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy PR to Preview
+env:
+  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches-ignore:    
+      - revert-logger
+    paths:
+      - "**.js"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: install Vercel CLI
+        run: npm install --global vercel@latest
+      - name: pull Vercel environment information
+        run: vercel pull --yes --environment=preview --token=${{ secrets.VERCEL_TOKEN }}
+      - name: build project artifacts
+        run: vercel build --token=${{ secrets.VERCEL_TOKEN }}
+      - name: deploy preview + assign beta domain
+        run: |
+          du --inodes -d 5 .vercel/output
+          ls -l .vercel/output/functions
+          vercel deploy --prebuilt --token=${{ secrets.VERCEL_TOKEN }} > domain.txt
+          vercel alias --scope ${{ secrets.VERCEL_TEAM_ID }} --token ${{ secrets.VERCEL_TOKEN }}
```

Comment:
What is this supposed to do?
It doesn't have one of the commands after alias like in the docs.
https://vercel.com/docs/cli/alias

Reply:
Oh, yes. I was previous doing a domain assigning linkfree-commitid.vercel.app then plan to keep it simple.

I have removed it. Thank you.